bundles/af-core/src/test/resources/form/textinput/test-content.json

```diff
@@ -93,7 +93,7 @@
     "readOnly": false,
     "autoComplete": "off",
     "label": {
-      "value": "Full Name"
```

Comment:
please add new collateral instead of changing existing collaterals. Applicable elsewhere as well.

Reply:
@ujjwal5 I have added new collateral as well but still we need to make changes in existing collateral since this collateral contains values according to the earlier behavior of AF v2 (Title gets overridden with name of the component on providing blank input). These test cases start to fail if the values don't get changed in collateral.